ydb/mvp/oidc_proxy/oidc_session_create_n.h

```diff
@@ -0,0 +1,208 @@
+#pragma once
+
+#include <util/generic/hash_set.h>
+#include <ydb/library/actors/core/actorsystem.h>
+#include <ydb/library/actors/core/actor.h>
+#include <ydb/library/actors/core/hfunc.h>
+#include <ydb/library/actors/core/events.h>
+#include <ydb/library/actors/core/event_local.h>
+#include <ydb/library/actors/core/actor_bootstrapped.h>
+#include <ydb/library/actors/http/http.h>
+#include <ydb/public/lib/deprecated/client/grpc_client.h>
+#include <ydb/library/grpc/client/grpc_client_low.h>
+#include <ydb/library/actors/core/log.h>
+#include <library/cpp/http/io/stream.h>
+#include <util/network/sock.h>
+#include <library/cpp/json/json_reader.h>
+#include <ydb/public/api/client/yc_private/oauth/session_service.grpc.pb.h>
+#include <ydb/mvp/core/protos/mvp.pb.h>
+#include <ydb/mvp/core/mvp_log.h>
+#include <ydb/mvp/core/mvp_tokens.h>
+#include <ydb/mvp/core/appdata.h>
+#include "openid_connect.h"
+
+namespace NMVP {
+
+class THandlerSessionCreateN : public NActors::TActorBootstrapped<THandlerSessionCreateN> {
```

Comment:
We have similar code to original `THandlerSessionCreate` here. I think that we need to reuse it and split into base and derived classes